sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -85,6 +86,25 @@ static rom_error_t rom_ext_verify(const manifest_t *manifest) {
                               &flash_exec);
 }
 
+/* These symbols are defined in
+ * `opentitan/sw/device/silicon_creator/rom_ext/rom_ext.ld`, and describe the
+ * location of the flash header.
+ */
+extern char _owner_virtual_start_address[];
+extern char _owner_virtual_size[];
+/**
+ * Compute the virtual address corresponding to the physical address `lma_addr`.
+ *
+ * @param manifest Pointer to the current manifest.
+ * @param lma_addr Load address or physical address.
+ * @return the computed virtual address.
+ */
+static uintptr_t owner_vma_get(const manifest_t *manifest, uintptr_t lma_addr) {
+  return (lma_addr - (uintptr_t)manifest +
```

Comment:
nit: Add TODO to harden this calculation as an attacker may be able to leverage a data store or bootstrap mechanism to insert malicious code.

Reply:
Created #13513 and added a TODO